ansible/files/bootstrap/telemetry.sh

```diff
@@ -62,6 +26,14 @@ while [ $# -gt 0 ]; do
             shift
             TIMEOUT=$1
             ;;
+        --email)
+            shift
+            EMAIL=$1
+            ;;
+        --efs-dns)
+            shift
+            EFS_DNS=$1
```

Comment:
@jezdez sorry for the delay in answering - it's not totally clear yet. It seems to me that even if someone is launching on the dev version of ATMO, they would still want to use the prod version of EFS. @vitillo, can you chime in here with thoughts you had on separate environments for EFS?